flutter/test/native_sdk_integration_test.dart

```diff
@@ -0,0 +1,163 @@
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+import 'package:sentry_flutter/src/version.dart';
+
+import 'mocks.dart';
+import 'sentry_flutter_options_test.dart';
+
+void main() {
+  group('$NativeSdkIntegration', () {
+    setUp(() {
+      TestWidgetsFlutterBinding.ensureInitialized();
+    });
+
+    test('test default values', () async {
+      String? methodName;
+      dynamic arguments;
+      final channel = createChannelWithCallback((call) async {
+        methodName = call.method;
+        arguments = call.arguments;
+      });
+      var sut = Fixture().getSut(channel);
```

Comment:
maybe create a new instance of the `Fixture` in the `setUp` as the other tests, so you don't have to repeat